diff --git a/Gemfile.lock b/Gemfile.lock
index a8b0746..ae0be69 100644
--- a/Gemfile.lock
+++ b/Gemfile.lock
@@ -125,6 +125,8 @@ GEM
     net-smtp (0.3.3)
       net-protocol
     nio4r (2.5.8)
+    nokogiri (1.14.2-arm64-darwin)
+      racc (~> 1.4)
     nokogiri (1.14.2-x86_64-darwin)
       racc (~> 1.4)
     parallel (1.22.1)
@@ -213,6 +215,7 @@ GEM
       actionpack (>= 5.2)
       activesupport (>= 5.2)
       sprockets (>= 3.0.0)
+    sqlite3 (1.6.1-arm64-darwin)
     sqlite3 (1.6.1-x86_64-darwin)
     stimulus-rails (1.2.1)
       railties (>= 6.0.0)
@@ -247,6 +250,7 @@ GEM
     zeitwerk (2.6.7)
 
 PLATFORMS
+  arm64-darwin-21
   x86_64-darwin-21
 
 DEPENDENCIES
diff --git a/app/controllers/appointments_controller.rb b/app/controllers/appointments_controller.rb
index e55b136..495d893 100644
--- a/app/controllers/appointments_controller.rb
+++ b/app/controllers/appointments_controller.rb
@@ -41,10 +41,10 @@ class AppointmentsController < ApplicationController
       sleep 1
       if @user.save
         update_slots_with_currency_and_user_id(@params, @user.id)
-        slot = Slot.find(@params[:slot_id])
+        @slot = Slot.find(@params[:slot_id])
         # UserMailer.notify(@user, slot).deliver_now
         # UserMailer.notify(@user).deliver_later(wait_until: 1.minute.from_now)
-        UserMailer.notify(@user, slot).deliver_later(wait_until: slot.slot_time + 165.minutes)
+        UserMailer.notify(@user, @slot).deliver_later(wait_until: @slot.slot_time + 165.minutes)
         format.turbo_stream do
           render turbo_stream: turbo_stream.update(:second_page, partial: 'appointments/congrats')
         end
diff --git a/app/views/appointments/_congrats.html.erb b/app/views/appointments/_congrats.html.erb
index 8f33a9d..c7f32a2 100644
--- a/app/views/appointments/_congrats.html.erb
+++ b/app/views/appointments/_congrats.html.erb
@@ -5,8 +5,8 @@
     <p style="font-size: x-large; font-weight: 300; margin-top: -10px;" >Appointment Booked</p>
     <div class="d-flex" style="margin-top: -10px; margin-left: 200px;">
       <%= image_tag("purple_calendar.svg") %>
-      <p style=" color: #542EBF; font-size: 22px; font-weight: 700; margin-top: 15px; margin-left: 10px;">
-        <%= Slot.find(@params[:slot_id]).slot_time.strftime("#{Slot.find(@params[:slot_id]).slot_time.day.ordinalize} %B, #{Slot.find(@params[:slot_id]).slot_time.in_time_zone("Chennai").strftime("%I:%M %p")}") %>
+      <p style=" color: #542EBF; font-size: 22px; font-weight: 700; margin-top: 15px; margin-left: 10px;">      
+        <%= @slot.slot_time.strftime("#{@slot.slot_time.day.ordinalize} %B, #{displayable_time(@slot.slot_time)}") %>
       </p>
     </div>
 
diff --git a/app/views/appointments/_slots.html.erb b/app/views/appointments/_slots.html.erb
index 80a9a2a..2dad5a5 100644
--- a/app/views/appointments/_slots.html.erb
+++ b/app/views/appointments/_slots.html.erb
@@ -6,7 +6,7 @@
 
   <div class="slot-buttons d-flex mt-2">
     <% @slots.each do |slot| %>
-      <% if slot.slot_time.in_time_zone("Chennai") < Time.now.in_time_zone("Chennai") %>
+      <% if in_ist(slot.slot_time) < in_ist(Time.now) %>
         <div>
           <button class="slot-btn btn btn-outline-secondary rounded-2 disable" disabled > <%= slot.slot_time.in_time_zone("Chennai").strftime("%I:%M %p") %> </button>
         </div>
diff --git a/config/database.yml b/config/database.yml
index fcba57f..6688221 100644
--- a/config/database.yml
+++ b/config/database.yml
@@ -11,15 +11,15 @@ default: &default
 
 development:
   <<: *default
-  database: db/development.sqlite3
+  database: db/doctor_appointment_development.sqlite3
 
 # Warning: The database defined as "test" will be erased and
 # re-generated from your development database when you run "rake".
 # Do not set this db to the same as development or production.
 test:
   <<: *default
-  database: db/test.sqlite3
+  database: db/doctor_appointment_test.sqlite3
 
 production:
   <<: *default
-  database: db/production.sqlite3
+  database: db/doctor_appointment_production.sqlite3
