diff --git a/app/views/appointments/_select_slot.html.erb b/app/views/appointments/_select_slot.html.erb
index fc18ee4..37e0108 100644
--- a/app/views/appointments/_select_slot.html.erb
+++ b/app/views/appointments/_select_slot.html.erb
@@ -41,7 +41,7 @@
   </div>
 
   <div class="mt-5">
-    <p class="high-demand-warning-text">Due to high demand, it is difficult to reschedule appointments. Please book your slot carefully!</p>
+    <p class="high-demand-warning-text"><%= t('appointments.select_slot.high_demand_warning_text')%></p>
     <%= link_to "Book Appointment", appointments_add_details_path, id: "book-appointment", "data-test-target": "bookAppointment", data: {turbo_stream: true, turbo_method: :get}, class: "btn btn-primary btn-lg rounded-2", disabled: true, style: "text-align: center; font-weight: 600; font-size: large; width: 100%; height: 56px; margin-right: 10px"%>
   </div>
 </div>
\ No newline at end of file
diff --git a/config/locales/en.yml b/config/locales/en.yml
index 8ca56fc..7725317 100644
--- a/config/locales/en.yml
+++ b/config/locales/en.yml
@@ -31,3 +31,7 @@
 
 en:
   hello: "Hello world"
+
+appointments:
+  select_slot:
+    high_demand_warning_text: "Due to high demand, it is difficult to reschedule appointments. Please book your slot carefully!"
\ No newline at end of file
